drizzlepac/haputils/align_utils.py

```diff
@@ -1345,19 +1345,47 @@ def update_image_wcs_info(tweakwcs_output, headerlet_filenames=None, fit_label=N
             cr2_comment = RMS_DEC_COMMENT.replace('mas', 'deg')
             hdulist[sci_extn].header['CRDER1'] = (info['RMS_RA'].deg, cr1_comment) if info['RMS_RA'] is not None else -1.0
             hdulist[sci_extn].header['CRDER2'] = (info['RMS_DEC'].deg, cr2_comment) if info['RMS_DEC'] is not None else -1.0
-            hdulist[sci_extn].header['NMATCHES'] = len(info['ref_mag']) if info['ref_mag'] is not None else 0
-            hdulist[sci_extn].header['FITGEOM'] = info['fitgeom'] if info['fitgeom'] is not None else 'N/A'
+            nmatch_val = len(info['ref_mag']) if info['ref_mag'] is not None else 0
+            hdulist[sci_extn].header['NMATCHES'] = nmatch_val
+            fitgeom_val = info['fitgeom'] if info['fitgeom'] is not None else 'N/A'
+            hdulist[sci_extn].header['FITGEOM'] = fitgeom_val
+
+            # adds rel keywords if they don't exist
+            keys_to_add = [
+                ("RELGEOM", "fitgeom for relative fit"),
+                ("RELMATCH", "number of matches for relative fit"),
+                ("RELRMS_D", "RMS in DEC of relative WCS fit(mas)"),
+                ("RELRMS_R", "RMS in RA of relative WCS fit(mas)"),
+                ("RELREFIM", "base reference image rootname for relative fit"),
+            ]
+            for keyword in keys_to_add:
+                if keyword[0] not in hdulist[sci_extn].header:
+                    hdulist[sci_extn].header.insert(
```

Comment:
Comments on your changes here:
(1) Inserting the keywords here was fine, it was just that the numeric keywords needed to have numeric values.  Since there are so few keywords to insert, a loop may or may not be useful because they are a mixture of strings and numeric values.
(2) Line 1364: The FITGEOM  and RELREFIM keywords are strings, so they should stay as "N/A".